packages/manager/apps/web/client/app/private-database/database/ram/update/private-database-database-ram-update.controller.js

```diff
@@ -115,19 +117,25 @@ angular.module('App').controller(
       this.loading.durations = true;
 
       this.privateDatabaseService
-        .getRamPrices(this.productId, {
-          ram: this.model.capacity,
-        })
-        .then(
-          (durations) => {
-            this.loading.durations = false;
-            this.data.durations.available = durations;
-          },
-          angular.noop,
-          (duration) => {
-            this.data.durations.available = duration;
-          },
-        );
+        .getRamPlan(this.productId, this.model.ram.planCode)
+        .then((order) => {
+          this.loading.durations = false;
+
+          const details = order.details.filter(({ detailType }) => {
+            return detailType === 'DURATION';
+          });
+
+          const { description } = details[0];
```

Comment:
```suggestion
          const [{ description }] = details;
```

Reply:
Should be simplified with `find` as only the first element is used

Reply:
done